### PR TITLE
Split graphs into separate rendering paths

### PR DESCRIPTION
**Pull request by danielnaab**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [ ] Update the changelog appropriately

Description
-----------
On-going work to split the well construction and water level charts into separate components that use the same JS component. This portion modifies state and corresponding selectors to support multiple components of the same type in the same document, by passing around an `options` object that includes a unique component ID.

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/components/graph/index.spec.js [hunk 1]
```diff
@@ -39,9 +39,9 @@ describe('graph component', () => {
             jasmine.Ajax.install();
             store = configureStore();
             attachToNode(store, div.node(), {
-                agencycode: 'USGS',
-                siteid: '430406089232901'
-            });
+                agencyCode: 'USGS',
+                siteId: '430406089232901'
+            }, 23);
```
Comment (mbucknell) [inline assets/src/scripts/components/graph/index.spec.js]: What is the 23 used for? The function only has three parameters?

Comment (danielnaab) [reply assets/src/scripts/components/graph/index.spec.js]: Originally I put the component ID as a separate argument, but missed this one. Fixed.

File: assets/src/scripts/components/graph/index.js [hunk 2]
```diff
@@ -31,24 +31,28 @@ export const drawMessage = function (elem, message) {
  * Draws a water-levels graph.
  * @param  {Object} store   Redux store
  * @param  {Object} node    DOM node to draw graph into
- * @param  {Object} options {agencycode, siteid} of site to draw
+ * @param  {Object} options {agencyCode, siteId} of site to draw
```
Comment (mbucknell) [inline assets/src/scripts/components/graph/index.js]: Should this be updated to reflect that an id is being passed in?

**Comment by danielnaab** [comment]
> General question: Where do the construction ids come from? I'm not seeing where the values get assigned.

The html table rows have it set here: https://github.com/ACWI-SOGW/ngwmn-ui/blob/master/server/ngwmn/templates/macros/components.html#L55 The ID is originally created in the `get_well_log` function in services/ngwmn.py

**Comment by mbucknell** [comment]
Thanks, I remember now.

**Pull request merged**